packages/vhd-lib/createStreamNbd.js

```diff
@@ -69,10 +69,33 @@ exports.createNbdVhdStream = async function createVhdStream(nbdClient, sourceStr
     }
   }
 
+  const totalLength = (offsetSector + blockSizeInSectors + 1) /* end footer */ * SECTOR_SIZE
+  let readedLength = 0
+  let lastProgress = 0
+  let lastUpdate = 0
+  function* trackAndYield(buffer) {
+    readedLength += buffer.length
+    const currentProgress = readedLength / totalLength
+    const formattedProgress = currentProgress.toFixed(2)
+    const now = Date.now()
+    const diffInSeconds = (now - lastUpdate) / 1000
+
+    if (lastProgress !== formattedProgress) {
```

Comment:
it will send an update any time there is a 0.1% change , I think it's too much

Reply:
0.1? 🤔 1% no?
`(1024 / 102400).toFixed(2) // 0.01 === 1%`